masonry/src/widget/label.rs

```diff
@@ -261,6 +261,10 @@ impl Widget for Label {
         SmallVec::new()
     }
 
+    fn make_trace_span(&self) -> Span {
+        trace_span!("Label")
```

Comment:
If this is necessary, is it worth just removing the default impl?

Reply:
It's not really necessary, it just makes better logs. I thought about removing the default impl, but the Widget trait is hard enough to implement as it is.